.github/workflows/quality_checks.yml

```diff
@@ -38,6 +38,14 @@ jobs:
         env:
           PYTHON_CONFIGURE_OPTS: --enable-shared 
 
+      - name: Install eps-spine-client package
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "//npm.pkg.github.com/:_authToken=${NODE_AUTH_TOKEN}" >> ~/.npmrc
+          echo "@nhsdigital:registry=https://npm.pkg.github.com" >> ~/.npmrc
+          npm install @nhsdigital/eps-spine-client
```

Comment:
shouldn't need to do this install, the make install below should do it for us

Reply:
The install has been removed from `Setting up .npmrc` step